File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/clis/CLIS2015U03ProtocolListInfo.java
package nta.med.data.model.ihis.clis;

public class CLIS2015U03ProtocolListInfo {
	private Integer clisProtocolId ;
	private String protocolCode    ;
	private String protocolName    ;
	public CLIS2015U03ProtocolListInfo(Integer clisProtocolId,
			String protocolCode, String protocolName) {
		super();
		this.clisProtocolId = clisProtocolId;
		this.protocolCode = protocolCode;
		this.protocolName = protocolName;
	}
	public Integer getClisProtocolId() {
		return clisProtocolId;
	}
	public void setClisProtocolId(Integer clisProtocolId) {
		this.clisProtocolId = clisProtocolId;
	}
	public String getProtocolCode() {
		return protocolCode;
	}
	public void setProtocolCode(String protocolCode) {
		this.protocolCode = protocolCode;
	}
	public String getProtocolName() {
		return protocolName;
	}
	public void setProtocolName(String protocolName) {
		this.protocolName = protocolName;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Clis/CLIS2015U03ProtocolListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Clis
{
    [Serializable]
    public class CLIS2015U03ProtocolListInfo
    {
        private String _clisProtocolId;
        private String _protocolCode;
        private String _protocolName;

        public String ClisProtocolId
        {
            get { return this._clisProtocolId; }
            set { this._clisProtocolId = value; }
        }

        public String ProtocolCode
        {
            get { return this._protocolCode; }
            set { this._protocolCode = value; }
        }

        public String ProtocolName
        {
            get { return this._protocolName; }
            set { this._protocolName = value; }
        }

        public CLIS2015U03ProtocolListInfo() { }

        public CLIS2015U03ProtocolListInfo(String clisProtocolId, String protocolCode, String protocolName)
        {
            this._clisProtocolId = clisProtocolId;
            this._protocolCode = protocolCode;
            this._protocolName = protocolName;
        }

    }
}